spark-jobs/src/main/scala/za/co/absa/enceladus/common/config/JobConfigParser.scala

```diff
@@ -51,7 +51,10 @@ object JobConfigParser {
   def jobConfigParser[R <: JobConfigParser[R]]: OParser[_, R] = {
     val builder = OParser.builder[R]
     import builder._
-    OParser.sequence(head("Job Parameters"),
+    OParser.sequence(
+
+      head(s"Enceladus ${this.getClass.getPackage.getImplementationVersion}"),
```

Comment:
```suggestion
      head(s"Enceladus ${this.getClass.getPackage.getImplementationVersion}v"),
```

or 

```suggestion
      head(s"Enceladus, v. ${this.getClass.getPackage.getImplementationVersion}"),
```

or something similar

Reply:
I'd be a fan of the latter.